services/amo/amo-base.js

```diff
@@ -21,7 +21,7 @@ class BaseAmoService extends BaseJsonService {
   async fetch({ addonId }) {
     return this._requestJson({
       schema,
-      url: `https://addons.mozilla.org/api/v3/addons/addon/${addonId}/`,
+      url: `https://addons.mozilla.org/api/v4/addons/addon/${addonId}/`,
```

Comment:
There were no breaking changes from v3 to v4 affecting any of the values we use on any of these badges